internal/collections/map.go

```diff
@@ -122,14 +123,14 @@ func (c *Map) Add(key string, value string) {
 
 // Set sets the value of a key with the array of strings passed. If the key already exists, it will be overwritten.
 func (c *Map) Set(key string, values []string) {
-	originalKey := key
```

Comment:
Here we are losing case sensitivenes, do you mind explaining why?

Reply:
I'm not entirely sure. That has been there since before, let me check why

Reply:
Oh just realised my mistake. Fixed that. Thank you!